### ISSUE TITLE
Feature Request: make the file types that show the count status configurable

### ISSUE THREAD

**Issue by megrxu** [holmescn/vscode-wordcount-cjk#1]
When dealing with TeX files, I need to run the command `Word Count` very frequently. And if I can configure which files that show the CJK word count status could be great.

For example, we can configure it in the `setting.json`:
```
{
  "wordcount_cjk.fileTypes": [
  "*.txt",
  "*.tex"
  ]
}
```

**Comment by holmesconan** [comment]
Thanks for your feature request. I will add this as soon as I can.

**Comment by holmesconan** [comment]
Sorry for the late response. I have published a new version: 1.0.1. But I don't add a configuration point for file type. I do not find a elegant method to support that. So a new trigger method is added:

If there is a `.editorconfig` file in current folder , the counter will be activated. EditorConfig is a config file that synchronize your editor configuration. It is useful whenever your cooperate with others. So it is not adding mess up your files.
